worlds/ladx/Locations.py

```diff
@@ -25,6 +25,17 @@
 def meta_to_name(meta):
     return f"{meta.name} ({meta.area})"
 
+def get_location_name_groups():
+    ret = {}
+    for s, v in checkMetadataTable.items():
+        if s == "None":
+            continue
+        if v.area not in ret:
+            ret[v.area] = []
+        ret[v.area].append(meta_to_name(v))
+    return ret

```

Comment:
```suggestion
def get_location_name_groups() -> Dict[str, Set[str]]:
    ret = {}
    for s, v in checkMetadataTable.items():
        if s == "None":
            continue
        ret.setdefault(v.area, set()).add(meta_to_name(v))
    return ret
```
It's supposed to be a Dict[str, Set[str]], not a Dict[str, List[str]]. Import `Dict` and `Set` from `typing` too.

`dict.setdefault` does the same thing as checking if something is in the dict and, if it's not, giving it a key and setting a default value.